frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -429,18 +431,45 @@ class Worksheet extends React.Component {
                     draggable: true,
                 });
             });
+            if (cmd_type === 'cut') {
+                // Remove the bundle lines
+                this.removeItemsFromSource(cutBundleIds);
```

Comment:
I think it's a bit confusing that we create and populate cutBundleIds even if cmd_type is not equal to "cut". How about if we only populate validBundles, and then change this line to:

```suggestion
                this.removeItemsFromSource(validBundles.map(e => e.id));
```